### PR TITLE
Update the Brazilian Portuguese translation to the August 1st, 2020 Version

### PR DESCRIPTION
**Pull request by mirandapedro**
This PR updates the Brazilian Portugues translation of the Regulations to the January 1st, 2020 Version of the Regulations.

Main changes include:
- Update of the Regulations added, removed or modified in between the 2018 and the 2020 versions of the Regulations;
- General textual revision and consistency improvment; and
- Creation of the Style Guide and the Contributors section in the README.md file.

### CODE REVIEW

**Comment by mirandapedro** [comment]
I'm not sure what is causing all of this "illegal characters" errors.

@thewca/wrc-team could you perhaps take a look at it?

Thanks.

**Comment by hanwu85** [comment]
All the trailing space are gone. Thanks!  
The illegal characters seems to focus on all the characters in specific lines.
I don't know why, I can't find any thing strange in the file.
Maybe we could just merge.  What do you think @lgarron ?

**Comment by Ivan-Brigidano** [comment]
@mirandapedro wouldn't it be better if you directly prepare a PR for the August 1st version?

**Comment by mirandapedro** [comment]
I believe this is now ready to be merged, if we agree to follow what @hanwu85 suggest about ignoring the illegal charecters Travis CI is highlighting.

**Comment by hanwu85** [comment]
That's my personal opinion. @lgarron ,could you share your thought on this issue?

**Comment by lgarron** [comment]
> ignoring the illegal charecters Travis CI is highlighting.

Hmm, I think we should first understand what's going on.

@viroulep, any chance I could ask you take take a quick look and see if it's a simple issue?

**Comment by viroulep** [comment]
Ignoring build errors cannot be an option!
It will simply not build the pdf, and break the build for everyone.

There were some pretty explicit error (like ["unexpected regulation line 478"](https://travis-ci.org/github/thewca/wca-regulations-translations/builds/714130836#L1250)) that I fixed.
The compiler also expects all lines to end with a `\n`, which was not the case for both files (and the cause of the issue for the illegal characters, as the "line" didn't match any of our patterns).
It could emit a much better error, I've created thewca/wca-regulations-compiler#18 for that.

**Comment by mirandapedro** [comment]
Thank you Phillipe!

@thewca/wrc-team are we ready to merge?

File: portuguese-brazilian/wca-guidelines.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # <wca-title>Orientações Adicionais da WCA
 
-<version>Versão: 1 de Janeiro de 2018
+<version>Versão: 1 de Agosto de 2020
 
 
 ## Notas
 
-Última revisão textual: 29 de Maio de 2018
+Última revisão textual: 1 de agosto de 2020
```
Comment (Ivan-Brigidano) [inline portuguese-brazilian/wca-guidelines.md:8]: Do you use capital letters for months in Portuguese? You have used them in the "Versão" but not in the "Última revisão textual".

Comment (mirandapedro) [reply portuguese-brazilian/wca-guidelines.md:8]: Thanks for spotting this! All of these should be in lower case. I'll make the necessary adjustments.

**Pull request merged**